examples/src/bin/wifi_embassy_access_point_with_sta.rs

```diff
@@ -9,8 +9,6 @@
 //!
```

Comment:
```suggestion
//! - alternatively, you can also connect to the same network and open the IP address printed by the example.
//!
```

Sorry, I was trying to say that something like this was missing now.

Reply:
Ah. Sorry about that. I have just woken up ;-)

Hopefully this is better now.